typed-protocols/typed-protocols.cabal

```
@@ -32,7 +32,7 @@ library
                    , TypeOperators
                    , BangPatterns
   build-depends:     base,
-                     io-classes >= 1.0 && < 1.4
+                     io-classes >= 1.0 && < 1.6
```

```suggestion
                     io-classes >= 1.4.1 && < 1.6
```
we're using `stToIO`